### DISCUSSION TITLE
degenerate characters in binary alignments

### DISCUSSION THREAD

**Discussion by rderelle** [iqtree/iqtree2#215] [Q&A] [answered]
Dear all,

I'm using IQtree2 with the binary model GTR2 and a binary alignment composed of '0' and '1', which indicate the presence of one indel variant or the other.

I would like to ask if the GTR2 model (or any other binary model) can handle degenerate characters (e.g., equivalent to degenerate nucleotides in DNA sequences). If yes, in which format should they be?

In my case, a degenerate character would indicate the presence of both variants 0 and 1.

Thanks,
Romain

**Comment by bqminh** [comment, marked as answer]
If you are not sure a position as 0 or 1, then you can put a "-" (gap) there. I don't see any problem of using GTR2 model on this data.
